lib/utils.js

```diff
@@ -95,10 +96,33 @@ function findClosestGitRepo(filePath) {
   return findClosestGitRepo(path.dirname(filePath));
 }
 
+// Returns a new Promise that resolve if the passed-in promise is rejected and
+// will be rejected with the provided error if the passed-in promise resolve.
+function reversePromise (promise, rejectionMessage) {
```

Comment:
may be worth adding a `resolveMessage` in this function signature to give the user more flexibility